onnxruntime/core/providers/dnnl/graph/dnnl_graph.cc

```diff
@@ -0,0 +1,254 @@
+// Copyright(C) 2022 Intel Corporation
```

Comment:
I suggest using dnnl_subgraph.h/*.cc instead of dnnl_graph since this is a subgraph of the full onnx graph.